### PR TITLE
[BB-90] Added initial styles to main page

### PR DESCRIPTION
**Pull request by cmills22**
**What ticket is this for?**

BB-90 https://eventmobi.atlassian.net/browse/BB-90

**What do we need to know to understand these changes?**

The changes made were the adding of styles which makes the main page of the to-do list conform to the event-mobi style-guide.

**What does this PR do?**

Styles the main page of the to-do list.

**How can we manually test your changes?**

Compare the page to the event mobi style-guide to ensure it follows the same style rules.

**Are there any blockers to merging this ticket?**

No

**What are you not sure of?**

N/A

**Relevant Documentation:**

Event mobi style-guide http://cms-style-guide.eventmobi.com/

### CODE REVIEW

File: body.css [hunk 1]
```diff
@@ -0,0 +1,250 @@
+/** 
+  * body.css
+  *
+  * These are the styles for the body of the webpage   
+  *
+  * Index
+  * - badge
```
Comment (mentosfreshmaker) [inline body.css]: what's the point of this comment?
ditto for all the other comments in css files

File: id.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
+/** 
+  * id.css
```
Comment (mentosfreshmaker) [inline id.css]: shouldn't organize css files by ids vs classes imo, but by functionality ie. all main layout css in one file, all modal css in one file. likely in alphabetical order? not sure how id vs class css is ordered, @mattfung99 ?

File: defaultStyles.css [hunk 3]
```diff
@@ -0,0 +1,34 @@
+/** 
+  * defaultStyles.css
```
Comment (mentosfreshmaker) [inline defaultStyles.css]: looks like naming css files should be all lowercase

**Review by mentosfreshmaker — CHANGES_REQUESTED** [review]

File: body.css (hunk 1 above)
Comment (mattfung99) [reply body.css]: @mentosfreshmaker This type of comment was written to give the reader a clear sense of what the spreadsheet style properties are used for in relation to the HTML page

File: defaultStyles.css (hunk 3 above)
Comment (mattfung99) [reply defaultStyles.css]: Will make changes accordingly

File: mainpage.css [hunk 4]
```diff
@@ -0,0 +1,192 @@
+/** 
+  * mainpage.css
```
Comment (jantnovi) [inline mainpage.css]: Do we need the file name up here?

Comment (mattfung99) [reply mainpage.css]: @jantnovi I don't think it is actually necessary or mandatory but from the CSS best practices research I've done, it is encouraged to do so. I'm not too sure actually.

File: todolist.html [hunk 5]
```diff
@@ -4,31 +4,35 @@
         <title>
             To-Do List
         </title>
+        <link rel='stylesheet' href='https://fonts.googleapis.com/css?family=Open+Sans:300'>
         <link rel='stylesheet' href='https://use.fontawesome.com/releases/v5.0.6/css/all.css'>
+        <link rel='stylesheet' href='body.css'>
+        <link rel='stylesheet' href='defaultStyles.css'>
```
Comment (mentosfreshmaker) [inline todolist.html]: update these

**Review by mentosfreshmaker — COMMENTED** [review]
Why are the rest of you not reviewing each others' PRs?

File: todolist.html (hunk 5 above)
Comment (mattfung99) [reply todolist.html]: Will do

**Review by mattfung99 — COMMENTED** [review]
I've made the changes to attaching the new CSS stylesheets onto the HTML

File: mainpage.css (hunk 4 above)
Comment (mentosfreshmaker) [reply mainpage.css]: eh seems redundant to me

File: mainpage.css [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+    background-color: #1e7e34;
+    border-color: #1e7e34;
+    color: #fff
+}
+.btn-success:not(:disabled):not(.disabled).active, .btn-success:not(:disabled):not(.disabled):active, .show > .btn-success.dropdown-toggle {
+    background-color: #1e7e34;
+    border-color: #1e7e34;
+    color: #fff
+}
+.btn-success:not(:disabled):not(.disabled).active:focus, .btn-success:not(:disabled):not(.disabled):active:focus,.show > .btn-success.dropdown-toggle:focus {
```
Comment (mentosfreshmaker) [inline mainpage.css]: so hard to read, break with newlines?

File: todoitem.css [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+.badge-danger[href]:focus, .badge-danger[href]:hover {
+    background-color: #bd2130;
+    color: #fff;
+    text-decoration: none;
+}
+.badge-dark {
+    background-color: #343a40;
+    color: #fff;
+}
+.badge-dark[href]:focus, .badge-dark[href]:hover {
```
Comment (mentosfreshmaker) [inline todoitem.css]: yea imo anything with more than one class/id, it gets hard to read without breaking it with newlines

Comment (mattfung99) [reply todoitem.css]: @mentosfreshmaker, I put each class into a new row in the cases where multiple properties are shared.

File: mainpage.css (hunk 4 above)
Comment (mattfung99) [reply mainpage.css]: Removed

**Review by cmills22 — COMMENTED** [review]
Everything looks good to me

File: mainpage.css (hunk 4 above)
Comment (mentosfreshmaker) [reply mainpage.css]: am I crazy or did you guys not remove these redundant names

File: todoitem.css [hunk 8]
```diff
@@ -0,0 +1,167 @@
...
+    margin: 0 2.5px 0 2.5px;
+    padding: 4.5px 20px;
+}
+.btn-edit {
+    background-color: #6ebb83;
+    color: #fff;
+    margin: 0 2.5px 0 2.5px;
+    padding: 4.5px 20px;
+}
+.my-4 {
```
Comment (mentosfreshmaker) [inline todoitem.css]: what is this

File: todolist.html [hunk 9]
```diff
@@ -4,31 +4,34 @@
...
         </title>
+        <link rel='stylesheet' href='https://fonts.googleapis.com/css?family=Open+Sans:300'>
         <link rel='stylesheet' href='https://use.fontawesome.com/releases/v5.0.6/css/all.css'>
+        <link rel='stylesheet' href='mainpage.css'>
+        <link rel='stylesheet' href='todoitem.css'> 
     </head>
     <body>
-        <div>
-            <img src='resources/EventMobiLogo.svg' alt='EventMobi'>
+        <div id='header'>
```
Comment (mentosfreshmaker) [inline todolist.html]: attributes use double quotes, update this file

File: todoitem.css [hunk 10]
```diff
@@ -0,0 +1,167 @@
...
+    margin-top: 1.5rem!important
+}
+.my-4 {
+    margin-bottom:1.5rem!important
+}
+.name {
+    font-size: 20px;
```
Comment (mentosfreshmaker) [inline todoitem.css]: you have a class just for the name? you can just do the font-size on the container

File: todolist.html [hunk 11]
```diff
@@ -4,32 +4,35 @@
...
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300">
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.6/css/all.css">
+        <link rel="stylesheet" href="mainpage.css">
+        <link rel="stylesheet" href="todoitem.css"> 
     </head>
     <body>
-        <div>
-            <img src='resources/EventMobiLogo.svg' alt='EventMobi'>
+        <div id="header">
+            <img class="logo" id="logo" src="resources/EventMobiLogo.svg" alt="EventMobi">
```
Comment (jantnovi) [inline todolist.html]: Why both class and id? The logo class does not map to anything in the css.

**Review by Raymond-Xia — COMMENTED** [review]
https://github.com/airbnb/css#formatting
Airbnb css style guide says to put blank lines between rule declarations to improve readability

**Review by mentosfreshmaker — APPROVED** [review]

**Comment by mentosfreshmaker** [comment]
Btw everyone who's reviewed should submit their approval first before merge

**Pull request merged**